README.md

```diff
@@ -8,8 +8,17 @@ access to filesystems, and network connectivity.
 
 Building
 ---
+There are a few methods for building the binary.
+_We use [go modules](https://github.com/golang/go/wiki/Modules), so you need to build with Go 1.11 or newer._
 
-We use [go modules](https://github.com/golang/go/wiki/Modules), so you need to build with Go 1.11 or newer. `go build` or `make` should be sufficient to generate a working firectl binary.
+#### Make
+`make` is a Makefile rule which executes `go build`
+
+`make docker` creates a temporary Docker container which builds and copies the binary to your current directory.
+#### Docker
```

Comment:
I think we can omit this section.  If someone wants to see how we use Docker, they can look at the definition of the `docker` target in the Makefile.

Reply:
Can you remove the entire "#### Docker" section?